tcms/testruns/tests/test_views.py

```diff
@@ -711,3 +711,32 @@ def test_change_assignee_without_permission(self):
         remove_perm_from_user(self.tester, 'testruns.change_testexecution')
         response = self.client.get(self.url)
         self.assertNotContains(response, self.change_assignee_html, html=True)
+
+class TestCaserunStatusOptions(BaseCaseRun):
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+        cls.url = reverse('testruns-get', args=[cls.test_run.pk])
+        cls.caserun_status_options_html = []
+
+        for tcrs in TestExecutionStatus.objects.all():
+            cls.caserun_status_options_html.append(
+                '<a value="{0}" href="#" class="{1}Blue9">{2}</a>'
+                .format(tcrs.pk, tcrs.name.lower(), tcrs.name)
+            )
```

Comment:
Notes:
class name should always be in English, otherwise the menu look horrible when translated. See `TestCaseExecution.icon()` method.

Reply:
I don't understand what i should do here.